### PR TITLE
Update all the readme

### PR DESCRIPTION
**Pull request by jeremi**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -129,10 +129,10 @@ repos:
     rev: v0.1.3
     hooks:
       - id: ruff
-        args: [--fix, --unsafe-fixes, --exit-non-zero-on-fix]
+        args: [--fix, --exit-non-zero-on-fix]
       - id: ruff-format
   - repo: https://github.com/OCA/pylint-odoo
-    rev: v8.0.19
+    rev: v9.0.4
```
Comment (jeremi) [inline .pre-commit-config.yaml]: Maybe we do not want to change this before the release. I fixed some of the issues, but not all.

**Review by gonzalesedwin1123 — APPROVED** [review]

**Review by reichie020212 — APPROVED** [review]

**Review by emjay0921 — APPROVED** [review]

**Review by kneckinator — CHANGES_REQUESTED** [review]
There are commits in this PR from @reichie020212 that we need to discuss.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -132,7 +132,7 @@ repos:
         args: [--fix, --exit-non-zero-on-fix]
       - id: ruff-format
   - repo: https://github.com/OCA/pylint-odoo
-    rev: v9.0.4
+    rev: v8.0.19
```
Comment (kneckinator) [inline .pre-commit-config.yaml:135]: @gonzalesedwin1123 what was the reason for reverting it?

Comment (gonzalesedwin1123) [reply .pre-commit-config.yaml:135]: There are pre-commit errors generated by the new version @kneckinator

Comment (kneckinator) [reply .pre-commit-config.yaml:135]: Are you ok with the rollback @jeremi ?

Comment (kneckinator) [reply .pre-commit-config.yaml:135]: I'll merge this "as-is" anyway. We can discuss the version later.

**Review by kneckinator — APPROVED** [review]
Looks good to me. Thanks @jeremi ! 🎉

**Pull request merged**